src/bmap.ml

```diff
@@ -23,47 +29,39 @@ module type S = sig
       [k] is removed from [m], and if the result is [Some v'] then
       [add k v' m] is returned. *)
 
-  val merge_safe :
+  val mergeSafe :
     f:(key -> [`Left of 'a | `Right of 'b | `Both of 'a * 'b] -> 'c option) ->
     'a t -> 'b t -> 'c t
-  (** [merge_safe ~f a b] merges the maps [a] and [b] together.
+  (** [mergeSafe ~f a b] merges the maps [a] and [b] together.
       @since 0.17 *)
 
-  val of_seq : (key * 'a) sequence -> 'a t
+  val ofList : (key * 'a) list -> 'a t
 
-  val add_seq : 'a t -> (key * 'a) sequence -> 'a t
+  val addList : 'a t -> (key * 'a) list -> 'a t
   (** @since 0.14 *)
 
-  val to_seq : 'a t -> (key * 'a) sequence
+ val toList : 'a t -> (key * 'a) list
 
-  val of_list : (key * 'a) list -> 'a t
+end
 
-  val add_list : 'a t -> (key * 'a) list -> 'a t
-  (** @since 0.14 *)
+module Make(O : Map.OrderedType) = struct
+  include Map.Make(O)
 
-  val keys : _ t -> key sequence
-  (** Iterate on keys only
-      @since 0.15 *)
+  let isEmpty = is_empty
 
-  val values : 'a t -> 'a sequence
```

Comment:
Are we sure we need a `valuesList`? Should we perhaps hold off on that? Since there isn't one in ocaml-containers, it doesn't seem to be too idiomatically terrible to leave it out for now.